tests/test_utils.py

```
@@ -0,0 +1,127 @@
+    sanitize_filename,
+    get_os_text,
+    get_next_number,
+)
+
+
+# ---------------------------------------------------------------------------
+# is_blueprint_file
+# ---------------------------------------------------------------------------
+
+class TestIsBlueprintFile:
+    EXTS = ['.pdf', '.dwg', '.dxf']
```

Ruff (0.15.10)

[warning] Mutable default value for class attribute

(RUF012)